### PR TITLE
Add latency measurement tests

### PR DESCRIPTION
**Pull request by fasaxc**
## Description

I built a bit of infrastructure to do the latency tests for network sets.  It's a bit rough around the edges but it seems sensible to get it checked in and running in semaphore so we can re-use it later.

## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: fv/latency_test.go [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+		})
+
+		It("with allow-all should have good latency", func() {
+			meanRtt := w[0].LatencyTo(w[1].IP, w[1].DefaultPort)
+			_, err := fmt.Fprintf(resultsFile, "allow-all: %v\n", meanRtt)
+			Expect(meanRtt).To(BeNumerically("<", 10*time.Millisecond))
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		Describe("with all() selector", func() {
```
Comment (neiljerram) [inline fv/latency_test.go]: Saying "source selector" would be a touch clearer, given that there's already another use of `all()` above.

File: fv/latency_test.go [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+				BeforeEach(func() {
+					w[1].WorkloadEndpoint.Spec.IPNetworks = append(w[1].WorkloadEndpoint.Spec.IPNetworks,
+						generateIPs(10000)...)
+					wep := w[1].WorkloadEndpoint
+					wep.Namespace = "fv"
+					_, err := client.WorkloadEndpoints().Update(utils.Ctx, wep, utils.NoOptions)
+					Expect(err).NotTo(HaveOccurred())
+
+					Eventually(func() int {
+						return getNumIPSetMembers(felix.Container, "cali4-s:5y5I3VdRZfDU01O--xXAPx2")
```
Comment (neiljerram) [inline fv/latency_test.go]: Is that IP set name predictable?

File: fv/latency_test.go [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+					}, "100s", "1000ms").Should(Equal(10002))
+				})
+
+				It("should have good latency", func() {
```
Comment (neiljerram) [inline fv/latency_test.go]: Perhaps worth commenting here that the `all()` source selector will now expand to an IP set with 10000+ entries.

File: fv/latency_test.go [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+				w[ii].Stop()
+			}
+		})
+	}
+
+	Context("IPv4: Network sets tests with initialized Felix and etcd datastore", func() {
+		describeLatencyTests(latencyConfig{ipVersion: 4})
+	})
+
+	// TODO: IPv6
```
Comment (neiljerram) [inline fv/latency_test.go]: Worth filling this in?  Does it just need an IPv6 version of generateIPs?

File: Makefile [hunk 5]
```diff
@@ -59,6 +59,7 @@ ifeq ($(ARCH),ppc64le)
 endif
 
 GO_BUILD_CONTAINER?=calico/go-build$(ARCHTAG):$(GO_BUILD_VER)
+FV_FELIXIMAGE?=calico/felix$(ARCHTAG)
```
Comment (neiljerram) [inline Makefile]: Should this have a `:latest` tag?

File: fv/latency_test.go (hunk 2 above)
Comment (fasaxc) [reply fv/latency_test.go]: Yes, it's a hash of the selector name with a fixed prefix.  Given this test isn't really about that, though, maybe I should calculate the value here.

**Pull request merged**